forte/data/data_store.py

```diff
@@ -135,36 +139,37 @@ def __init__(self):
         """
         self.entry_dict: dict = {}
 
-    def _new_annotation(self, type: str, begin, end):
-        """
-        generate a new annotation with default fields. Called by add_entry_raw()
-        to create a new annotation with `type`, `begin`, and `end`.
+    def _new_annotation(self, entry_type: str, begin: int, end: int):
+        r"""This function generates a new annotation with default fields.
+        Called by add_annotation_raw() to create a new annotation
+        with `type`, `begin`, and `end`.
 
         Args:
-            type (str): type of this annotation
+            entry_type (str): Fully qualified name of this annotation.
+            begin (int): Begin index of the entry.
+            end (int): End index of the entry.
+
         Returns:
-            A list representing a new annotation type entry data
+            A list representing a new annotation type entry data.
         """
 
         tid: int = uuid.uuid4().int
-        entry = [begin, end, tid, type]
+        entry = [begin, end, tid, entry_type]
         entry += len(self._type_attributes[type]) * [None]
```

Comment:
type? I thought you change this to entry_type

Reply:
yes forgot to change here